scripts/drone/steps/lib.star

```diff
@@ -475,6 +475,7 @@ def lint_frontend_step():
         'commands': [
             'yarn run prettier:check',
             'yarn run lint',
+            'yarn run i18n:compile', # TODO: right place for this?
```

Comment:
@grafana/grafana-release-eng I would appreciate a peek at this if you have time 🥺 

The frontend now imports some auto-generated JS files (built with `yarn run i18n:compile`). `yarn build` and `yarn start` (for dev) will build these files for the app build, but they're not present for this lint step.

Is this the right place to include the step? Do you have other thoughts on this?